porter/v1/cluster_control_plane.proto

```diff
@@ -240,6 +240,23 @@ message UpdateRevisionStatusRequest {
 
 message UpdateRevisionStatusResponse {}
 
+message RollbackRevisionRequest {
+  int64 project_id = 1;
```

Comment:
comment for project_id?

Reply:
we haven't been adding a comment for project_id so just maintaining consistency I guess